crates/uv/src/commands/project/run.rs

```diff
@@ -1238,10 +1253,22 @@ impl RunCommand {
     }
 
     /// Convert a [`RunCommand`] into a [`Command`].
-    fn as_command(&self, interpreter: &Interpreter) -> Command {
+    fn as_command(&self, interpreter: &Interpreter, is_patch_request: bool) -> Command {
         match self {
             Self::Python(args) => {
-                let mut process = Command::new(interpreter.sys_executable());
+                let mut process = if is_patch_request || !interpreter.is_standalone() {
+                    Command::new(interpreter.sys_executable())
+                } else {
+                    // If this is a standalone interpreter, there is no patch request, and a
```

Comment:
Maybe a dumb question, but.. why isn't `sys_executable` the executable in the symlink directory?

Reply:
For a specific interpreter, that would potentially be inaccurate. The symlink directory-containing executable path only points to the highest patch version. So I don't know if it makes sense to have that on `Interpreter` since an interpreter doesn't know anything about other interpreters.

Reply:
Ah I'm not asking why the logic isn't in `Interpeter`, I'm asking why the interpreter itself isn't reporting `sys.executable` as the file in the symlink directory.

Reply:
I am just confused that we need to special-case `uv run`. Doesn't `python -m venv` work by itself?

Reply:
So `sys.executable` will give the real/resolved path to the executable in the symlink directory. The `venv` module uses the actual path it was called with to populate the `pyvenv.cfg:home` key and to determine the target path for the symlink in `.venv/bin` (on Unix). So this special-casing is ensuring we call it with the symlink directory path instead of the fully resolved one.

Reply:
Is the key here is that `Interpreter` for these managed installations is being queried from the resolved directory instead of the symlinked one?